libfs/remote_status.go

```diff
@@ -109,7 +107,7 @@ func (r *RemoteStatus) ExtraFileName() string {
 	r.Lock()
 	defer r.Unlock()
 
-	if r.extraFileName == "" || time.Since(r.failingSince) > failureDisplayThreshold {
+	if r.extraFileName == "" || time.Since(r.failingSince) < failureDisplayThreshold {
```

Comment:
We return "" (= no error) if either:
+ There is no error (r.extraFilename == "")
+ There is an error but it might be a flake

Only start erroring out when there is a stable error that has presisted for failureDisplayThreshold.